subsys/bluetooth/host/settings.c

```diff
@@ -240,6 +240,13 @@ static int commit(void)
 		bt_set_name(CONFIG_BT_DEVICE_NAME);
 	}
 #endif
+
+#if defined(CONFIG_BT_DEVICE_NAME_DYNAMIC)
+	if (!bt_dev.id_count) {
+		(void) bt_set_appearance(CONFIG_BT_DEVICE_APPEARANCE);
+	}
```

Comment:
```suggestion
	if (!bt_dev.id_count) {
		(void)bt_set_appearance(CONFIG_BT_DEVICE_APPEARANCE);
	}
```
or add proper check of return value. 

I wonder why we don't have an error check for `bt_set_name` in this context.

Reply:
The error is already logged by logged in `bt_set_appearance`. There is nothing else to do in the error handling. Not sure if I should move out the logging?